tutorials/continuous-aggs-tutorial.md

```diff
@@ -0,0 +1,238 @@
+
+# Tutorial: Continuous Aggregates
+
+One kind of query that occurs frequently when performing data analysis is
+aggregating data into summaries.
+
+For instance if we want to find out the average of a column for each day in the 
+dataset stored in a hypertable, we would run 
+
+``` sql
+SELECT time_bucket('1 day', timedimesion-column-name), avg(column-name), stddev(column-name)
```

Comment:
The dashes in the arguments make the syntax highlighting look odd.  See attached
![Screen Shot 2019-04-26 at 3 22 48 PM](https://user-images.githubusercontent.com/12288273/56831624-8ed00c00-6837-11e9-92e5-5666be4d6eff.png)

Reply:
modified.